recipes/s_gd2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "s_gd2" %}
+{% set version = "1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bc69e36bac679dd968056dce3aaf5d3ff107b2ea4ff99364791ed7f8246807b0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.16.0
```

Comment:
Please see https://conda-forge.org/docs/maintainer/knowledge_base.html?#building-against-numpy for linking against numpy

Reply:
Is this right?

Reply:
Yep!